x-pack/plugin/security/qa/multi-cluster/src/javaRestTest/java/org/elasticsearch/xpack/remotecluster/RemoteClusterSecurityEsqlIT.java

```diff
@@ -491,6 +495,10 @@ public void testCrossClusterQueryWithRemoteDLSAndFLS() throws Exception {
         assertThat(flatList, containsInAnyOrder("engineering"));
     }
 
+    /**
+     * Note: invalid_remote is "invalid" because it has a bogus API key, but the cluster does actually exist
```

Comment:
Does `invalid_remote` also always have `skip_unavailable=true`? Because as I understand if it's not, it should error out?

Reply:
Good catch. I think my understanding of this test is wrong. I'll work on it next and update the comments and make sure this is behaving as expected.

Reply:
I've updated the javadoc comment. The test is actually missing and we get different errors from the security layer depending on its skip_un=true/false flag, which explains the different behavior in the test seen below (for the remote-only test; the local+remote test has a different issue).